Update sample-v2-code.md

premgan opened the pull request:
Added Microsoft Teams App sample leveraging MSAL.

Commit:
Update sample-v2-code.md

Added Microsoft Teams App sample leveraging MSAL.

mmacy commented on articles/active-directory/develop/sample-v2-code.md:
```diff
@@ -136,6 +136,15 @@ The following sample shows a public client application running on a device witho
 
+## Microsoft Teams applications
+
+The following samples illustrate Microsoft Teams applications that sign in users. Some samples also demonstrate how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).
```
Since there's only one sample, reword this to be more descriptive of the current contents and ideally support future additions to the table (not everyone will remember to update this intro).

If there will be only tabs apps, for example, and they're all going to be multi-platform (web, native, mobile, et al.), add that information to this intro paragraph.

mmacy replied:
@premgan bump on this review.

premgan replied:
@mmacy - I should be able to complete this week, Was working on the Meta data piece first.

mmacy commented on articles/active-directory/develop/sample-v2-code.md:
```diff
@@ -136,6 +136,15 @@ The following sample shows a public client application running on a device witho
+> [!div class="mx-tdCol2BreakAll"]
+> | Language/<br/>Platform | Code sample(s) <br/>on GitHub | Auth<br/> libraries | Auth flow |  
+> | ------- | -------- | ------------- | -------------- |  
+> | Node | [Teams Tab App - Single Sign on and call Microsoft Graph via OBO](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-sso/nodejs) | MSAL Node |  On-Behalf-Of (OBO) |
```
```suggestion
> | Node.js | [Teams Tab app: single sign-on (SSO) and call Microsoft Graph](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-sso/nodejs) | MSAL Node |  On-Behalf-Of (OBO) |
```

- add .js
- added "SSO" to help those who CTRL+F on the page and search for _SSO_
- token grant flow has its own column

mmacy reviewed (changes requested):
Couple of required changes requested, @premgan.

Commit:
Update articles/active-directory/develop/sample-v2-code.md

Co-authored-by: Marsh Macy <mmacy@users.noreply.github.com>

Commit:
Update sample-v2-code.md

mmacy commented on articles/active-directory/develop/sample-v2-code.md:
```diff
@@ -136,6 +136,15 @@ The following sample shows a public client application running on a device witho
 
+## Microsoft Teams applications
+
+The following sample illustrates Microsoft Teams Tab application that signs in users. Additionally it demonstrates how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).
```
```suggestion
This is a Microsoft Teams Tab application that signs in users and calls a web API (Microsoft Graph or your own API) with the user's identity.
```

mmacy replied:
@premgan Oh, and this is good for merge if you take the latest suggestion here _and_ the sample does indeed demonstrate how to use your own web API as the downstream API. Thanks!

premgan replied:
> @premgan Oh, and this is good for merge if you take the latest suggestion here _and_ the sample does indeed demonstrate how to use your own web API as the downstream API. Thanks!

I removed the reference to calling your own Web API and just scoped to Graph API.

mmacy reviewed (changes requested):
@premgan Can you point me to where in the code it demonstrates how to call your own (custom) web API and not Graph?

Commit:
Update sample-v2-code.md

Removed reference to calling your own API since code only shows how to call Graph API.

mmacy reviewed (approved):
LGTM! Thanks for all the edits, @premgan. :shipit:
